COLLABORATOR_GUIDE.md

```diff
@@ -587,7 +587,7 @@ $ git push upstream master
 
 Close the pull request with a "Landed in `<commit hash>`" comment. If
 your pull request shows the purple merged status then you should still
-add the "Landed in <commit hash>..<commit hash>" comment if you added
+add the "Landed in \<commit hash>..\<commit hash>" comment if you added
```

Comment:
This is an improvement for sure (because now it actually renders) so 👍 as-is, but FWIW, it might be good to get the code-display highlighting (`like this`) from line 588 on this line too or else to remove it from line 588. Either one, but consistent.

Reply:
Were you thinking like this:
```markdown
add the "Landed in `<commit hash>`..`<commit hash>`" comment if you added
```
or
```markdown
add the "Landed in `<commit hash>..<commit hash>`" comment if you added
```

Reply:
Not sure, honestly. Maybe just omitting the backticks entirely?

Reply:
Looks like it landed with just the escape. BTW, the backticks in the examples above were just because I codefence'd the line as markdown to show placement of the backticks as where to start/end the code sections that you were talking about

Reply:
> the backticks in the examples above were just because I codefence'd the line as markdown 

Yes, I understand. I was saying that maybe it makes the most sense to eliminate code sections entirely in that bit because `<commit hash>..<commit hash>` isn't really code. It's unclear whether to also treat `Landed in` as code, etc. etc. So, just treat it all like a string. But this is all pretty minor stuff and I'm OK with it not being fixed at all or else being fixed in whatever way someone else wants as long as the end result is consistency.

Reply:
Sounds good, I'm going to leave it since it's rendering again :)